lib/engine/ability/blocks_division.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+require_relative 'base'
+
+module Engine
+  module Ability
+    class BlocksDivision < Base
+      attr_reader :division_type
```

Comment:
just call it type

Reply:
`type` is already used for the type of the ability